app/experimenter/docs/swagger-ui.html

```diff
@@ -3291,6 +3291,9 @@
                 },
                 "schema": {
                   "type": "string"
+                },
+                "setsPrefs": {
+                  "type": "boolean"
```

Comment:
`setsPrefs`?

Reply:
Yeah that's odd, I ran `make generate_docs generate_types` so it's weird that that was missed in @brennie 's PR, it should fail a check if there's pending updates to docs/types like that.  Let me try rerunning it.

Reply:
Yeah see look, it's missed in main:

```
[app-9BVIRsKI-py3.9]jaredlockhart@Jareds-MacBook-Pro~/Documents/experimenter(main)$ git diff
diff --git a/app/experimenter/docs/openapi-schema.json b/app/experimenter/docs/openapi-schema.json
index 3615988e..598165f1 100644
--- a/app/experimenter/docs/openapi-schema.json
+++ b/app/experimenter/docs/openapi-schema.json
@@ -3649,4 +3649,4 @@
       }
     }
   }
-}
+}
\ No newline at end of file
diff --git a/app/experimenter/docs/swagger-ui.html b/app/experimenter/docs/swagger-ui.html
index 13a4cb59..2381fc83 100644
--- a/app/experimenter/docs/swagger-ui.html
+++ b/app/experimenter/docs/swagger-ui.html
@@ -3291,6 +3291,9 @@
                 },
                 "schema": {
                   "type": "string"
+                },
+                "setsPrefs": {
+                  "type": "boolean"
                 }
               },
               "required": [
@@ -3300,7 +3303,8 @@
                 "description",
                 "application",
                 "ownerEmail",
-                "schema"
+                "schema",
+                "setsPrefs"
               ]
             }
           },
[app-9BVIRsKI-py3.9]jaredlockhart@Jareds-MacBook-Pro~/Documents/experimenter(main)$ 
```

Reply:
Filed https://github.com/mozilla/experimenter/issues/7911

Reply:
thank you for your prompt response 🚀